packages/angular-test/src/boolean.ts

```diff
@@ -205,7 +204,7 @@ export const booleanBaseTest = <C extends JsonFormsControl, I>
         mockSubStore.complete();
         fixture.detectChanges();
         component.ngOnInit();
-        expect(checkboxInstance.disabled).toBe(true);
+        expect(component.form.disabled).toBe(true);
```

Comment:
imho we must check the checkbox instance and not the component, see comment on the renderer